Library/Homebrew/blacklist.rb

```diff
@@ -1,16 +1,5 @@
 def blacklisted? name
   case name.downcase
-  when 'screen', /^rubygems?$/ then <<-EOS.undent
```

Comment:
Yeah, you are right. Maybe we should handle it the way other around. That is to inform users the tools can be found in OS X when searching taps return the result of homebrew/dupes. 

Overall, I think we should offer a better user experience and not all of users are aware what `dupes` means in the context of `homebrew/dupes`